doc/flame/effects.md

```diff
@@ -551,18 +577,18 @@ common controllers. The syntax of the constructor is the following:
 
 ```dart
 EffectController({
-    required double duration,
-    Curve curve = Curves.linear,
-    double? reverseDuration,
-    Curve? reverseCurve,
-    bool alternate = false,
-    double atMaxDuration = 0.0,
-    double atMinDuration = 0.0,
-    int? repeatCount,
-    bool infinite = false,
-    double startDelay = 0.0,
-    VoidCallback? onMax,
-    VoidCallback? onMin,
+  required double duration,
+  Curve curve = Curves.linear,
+  double? reverseDuration,
+  Curve? reverseCurve,
+  bool alternate = false,
+  double atMaxDuration = 0.0,
+  double atMinDuration = 0.0,
+  int? repeatCount,
+  bool infinite = false,
+  double startDelay = 0.0,
+  VoidCallback? onMax,
+  VoidCallback? onMin,
```

Comment:
I didn't understand the need of changing all these (not only these but many from this file). It is better to only change that are needed.

Or, is there any reason for the changes